drafts/developer-guide/pages/data-privacy/index.adoc

```diff
@@ -0,0 +1,123 @@
+include::ROOT:partial$variables.adoc[]
+
+= Data privacy
+
+Data privacy allows us to configure and describe how and where sensitive data is stored on a module-field level.
+
+It expands the current data storing capabilities and adds features that enable {PRODUCT_NAME} administrators to have more control over data privacy and managing private data.
+
+=== *Defining Privacy Policies*.
+
+With information from data sources set on a specific data module, each record will be able to hold detailed information about the ownership and location(s) of particular data. 
+
+System Administrators will be obliged to provide a data storage location upon setting a Corteza data module. 
+
+{PRODUCT_NAME} will enable the selection of predefined data policies or new policies to describe the processing purpose and rules at a record field level.
+
+== Sensitivity Levels
```

Comment:
The sensitivity level can also be defined on connections, so this should be added similarly to how you've handled modules and module fields.

Reply:
Please explain